pandas/core/indexes/multi.py

```diff
@@ -1263,6 +1263,11 @@ def remove_unused_levels(self):
 
         .. versionadded:: 0.20.0
 
+        Parameters
```

Comment:
yes i am sure
those are terrible patterns for an immutable object and should be removed